snuba/datasets/transactions_processor.py

```diff
@@ -248,6 +248,12 @@ def _process_contexts_and_user(
             if context in contexts:
                 del contexts[context]
 
+        appContext = contexts.get("app")
+        if appContext is not None:
+            appStartType = appContext.get("start_type")
+            if appStartType is not None:
+                processed["app_start_type"] = appStartType
```

Comment:
Snuba will implicitly populate a `None` value for any missing columns, but I would rather be as explicit as possible for clarity's sake. Can you set `processed["app_start_type"]` to `None` by default?

Reply:
@evanh, I can confirm that setting `processed["app_start_type"] = None` fails the tests. I don't know how to fix this, as I have no clue how Snuba works, and I also don't have Python knowledge. Can't I remove it here? Do I need to change things somewhere else?